packages/features/workspace_screen/lib/src/components/bottom_bar/bottom_nav_bar.dart

```diff
@@ -5,32 +5,49 @@ import 'package:l10n/l10n.dart';
 import 'package:tools/tools.dart';
 import 'package:workspace_screen/workspace_screen.dart';
 
-class BottomNavBar extends StatelessWidget {
+class BottomNavBar extends ConsumerWidget {
```

Comment:
I would not extend the ConsumerWidget
Usually you wrap the widget with a Consumer Widget and then you get a variable that is your instance of the reference you need. You can then hand this over to the function.